core/snippets/snippets/forLoopStatement.snippet

```diff
@@ -22,3 +22,10 @@ language: python
 for i in range($1):
     $0
 ---
+
+language: r
+-
+for (${1:x} in 1:${2:10}) {
```

Comment:
Why `10` as a placeholder?

Reply:
that's the numerical value to range up to, same as the x in python's range(x)